tests/database/queries/activities.test.js

```diff
@@ -1,91 +1,110 @@
 const test = require('tape');
+const pg = require('pg');
+const { getConfig } = require('../../../config');
+const { refresh: refreshDB, empty: emptyDB } = require('../../../db/scripts');
 const activities = require('../../../react-backend/database/queries/activities');
 const activitiesForToday = require('../../../react-backend/database/queries/activities_today');
 const deleteActivity = require('../../../react-backend/database/queries/activity_delete');
 const insertActivity = require('../../../react-backend/database/queries/activity_add');
 const updateActivity = require('../../../react-backend/database/queries/activity_update');
 
-const rebuild = require('../../../react-backend/database/database_rebuild');
-const clean = require('../../../react-backend/database/database_clean');
+const config = getConfig(process.env.NODE_ENV);
 
 test('Tests activities responds with the correct data', async (t) => {
-  await rebuild();
+  await refreshDB();
 
-  const activityArray = await activities(2);
+  const client = new pg.Client(config.psql);
```

Comment:
We don't need to run tests with pg.Client. I've fixed the test bug with not being able to shut down the pool in my local. There was an unhandled promise that wasn't being returned hidden away.

Reply:
Ignore 😊